utils/create_command_docs.py

```diff
@@ -89,7 +89,7 @@ def get_cmd_metadata(
         options=[param for param in params if param.param_type_name == "option"],
         parent_reference=parent_reference,
         subcommands=subcommands,
-        usage=cmd.get_usage(context),
+        usage="\n".join(map(lambda x: x[7:], cmd.get_usage(context).split("\n"))),
```

Comment:
Not a fan of comments, but it might be worth adding something around this magical `7` number.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/170/commits/290b76d0e3d875e82bcd1ac761ebf98ab69f2c3f.